### ISSUE TITLE
Firefox Extension

### ISSUE THREAD

**Issue by ceticdev** [tmxkn1/Microsoft-Reward-Chrome-Ext#81]
I know this is meant to be a Chrome extension, but since this plugin will eventually have to be updated for manifest v3, could a Firefox version of this be ported? I know lots of people don't like the new upcoming manifest changes so this would be a great addition.

Thanks for the hard work! Using selenium for Rewards is so much more annoying.

**Comment by rokejulianlockhart** [comment]
[It exists.](http://addons.mozilla.org/firefox/addon/bing-autosearch)

**Comment by tmxkn1** [comment]
It seems to be a direct port of my work but that's fine :)

**Comment by rokejulianlockhart** [comment]
@tmxkn1, in which case, since the work has already been done, why not upstream the work or rebase to it?

After all, if a 3rd-party is maintaining it, if they decide to cease to, or include harmful code, your reputation shall be negatively affected rather than theirs.

**Comment by tmxkn1** [comment]
> @tmxkn1, in which case, since the work has already been done, why not upstream the work or rebase to it?
> 
> After all, if a 3rd-party is maintaining it, if they decide to cease to, or include harmful code, your reputation shall be negatively affected rather than theirs.

Fair point. I will publish it on FF store once I update it with v3 manifest.

**Comment by kaminki** [comment]
Hi there,
I'm the guy who uploaded that extension.
I'm not good with javascript, (only recently started) I only uploaded it for convenience of others, so don't worry about malicious code.
Once OP uploads an official extension I will take it down as to not drag attention away from the creator.

**Comment by NelsonH75** [comment]
Side thing: When this does get implemented, it would be great if it worked with Firefox containers.
